apps/studio/components/layouts/ProjectLayout/LayoutHeader/NotificationsPopover/NotificationRow.tsx

```diff
@@ -3,18 +3,18 @@ import dayjs from 'dayjs'
 import { Button, IconX } from 'ui'
 
 import { useNotificationsDismissMutation } from 'data/notifications/notifications-dismiss-mutation'
+import { useOrganizationsQuery } from 'data/organizations/organizations-query'
+import { Project } from 'data/projects/project-detail-query'
 import { useProjectsQuery } from 'data/projects/projects-query'
 import { useStore } from 'hooks'
-import { Project } from 'types'
 import NotificationActions from './NotificationActions'
 import { formatNotificationCTAText, formatNotificationText } from './NotificationRows.utils'
-import { useOrganizationsQuery } from 'data/organizations/organizations-query'
 
 export interface NotificationRowProps {
   notification: Notification
-  onSelectRestartProject: (project: Project, notification: Notification) => void
-  onSelectApplyMigration: (project: Project, notification: Notification) => void
-  onSelectRollbackMigration: (project: Project, notification: Notification) => void
+  onSelectRestartProject: (project: any, notification: Notification) => void
+  onSelectApplyMigration: (project: any, notification: Notification) => void
+  onSelectRollbackMigration: (project: any, notification: Notification) => void
```

Comment:
Feels like the TS went backwards here – can we use ProjectInfo now or something similar?